dbt/node_runners.py

```diff
@@ -61,6 +62,26 @@ def skipped(self):
         return self.skip
 
 
+class RunOperationResult(RunModelResult):
+    def __init__(self, node, error=None, skip=False, status=None,
+                 failed=None, execution_time=0, returned=None):
+        super(RunOperationResult, self).__init__(node, error, skip, status,
+                                                 failed, execution_time)
+        self.returned = returned
+
+    @property
+    def errored(self):
+        return self.error is not None
+
+    @property
+    def failed(self):
+        return self.fail
+
+    @property
+    def skipped(self):
+        return self.skip
```

Comment:
it looks to me like you don't need to override `errored`, `failed`, or `skipped`